config.js

```diff
@@ -3,8 +3,11 @@ require('dotenv').config()
 module.exports = {
   before (context) {
     const standalone = process.env.SELENIUM_STANDALONE
+    const appium = process.env.APPIUM
     if (standalone) {
       context.webdriver.standalone = standalone
+    } else if (appium && !standalone) {
```

Comment:
You don't need to add the `!standalone` condition since this is else if and there is a `standalone` condition before it.